CODEOWNERS

```diff
@@ -35,12 +35,15 @@
 /src/lib/mina_numbers/ @MinaProtocol/crypto-eng-reviewers
 /src/lib/crs/ @MinaProtocol/crypto-eng-reviewers
 /src/lib/crypto_params/ @MinaProtocol/crypto-eng-reviewers
+/src/lib/crypto @MinaProtocol/crypto-eng-reviewers
+/src/lib/crypto/kimchi_bindings/js @MinaProtocol/crypto-eng-reviewers @MinaProtocol/product-eng-reviewers
```

Comment:
Should this be all of product eng, or just you @mitschabaude?

Reply:
Good question :) I can reduce it to me for now to make it more tight, and add other JS devs when they start contributing to those parts of the code base